### PR TITLE
Loading file icons from the bundle

### LINKED ISSUE #89: File icons should work on Firefox
Screenshot 2020-01-07 at 14 55 15

From the Browser console:
> Content Security Policy: The page’s settings blocked the loading of a resource at cdn.jsdelivr.net/npm/file-icons-js@1.0.3/css/style.css (“style-src”).

### PR DESCRIPTION
**Pull request by adambriny**
In this PR:

- Fixed #89 
- Made file icons loaded from the bundle rather than downloaded from a cdn.

### CODE REVIEW

**Comment by berzniz** [comment]
Thanks @adambriny. That's a great contribution, I'll test and merge.

File: src/js/lib.js [hunk 1]
```diff
@@ -175,6 +175,14 @@ export const isElementVisible = (el) => {
 
 const EMPTY_FILTER = ''
 
+export const getBrowserApi = () => {
```
Comment (berzniz) [inline src/js/lib.js:178]: Nice 👏

File: src/js/index.jsx [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
 import { render } from 'react-dom'
 import Tree from './components/tree'
-import { createFileTree, createRootElement } from './lib'
+import { createFileTree, createRootElement, getBrowserApi } from './lib'
+
 import './style.css'
 
+__webpack_public_path__ = getBrowserApi().runtime.getURL('')
```
Comment (berzniz) [inline src/js/index.jsx:8]: Why is `__webpack_public_path__` necessary? It seems to be working correctly without it

File: src/js/lib.js (hunk 1 above)
Comment (adambriny) [reply src/js/lib.js:178]: Thanks! :bow:

File: src/js/index.jsx (hunk 2 above)
Comment (adambriny) [reply src/js/index.jsx:8]: Indeed it is working. It seems like the file-loader update was enough to make it handle the font paths correctly. 
But with the old version, it's looking for the fonts on `https://github.com/<pr-path>/fonts/...`

Comment (adambriny) [reply src/js/index.jsx]: Removed in 902b53c

**Pull request merged**